app/src/main/java/ch/epfl/sdp/location/ConcreteLocationBroker.java

```diff
@@ -83,32 +81,6 @@ public boolean hasPermissions(Provider provider) {
 
     @Override
     public void requestPermissions(Activity activity, int requestCode) {
-        // TODO: uncomment this
-        /* This is the correct implementation. Could not test it!
```

Comment:
Wasn't supposed to be the correct implementation?

Reply:
Ahahahah yes, but that code was hard to test